homework/src/index.js

```diff
@@ -30,17 +33,108 @@
     return elem;
   }
 
-  function main(url) {
-    fetchJSON(url, (err, data) => {
-      const root = document.getElementById('root');
-      if (err) {
-        createAndAppend('div', root, { text: err.message, class: 'alert-error' });
-      } else {
-        createAndAppend('pre', root, { text: JSON.stringify(data, null, 2) });
-      }
+  function renderRepository(repository, container) {
+    container.innerHTML = '';
+    const table = createAndAppend('table', container);
+    const tbody = createAndAppend('tbody', table);
+    const tr1 = createAndAppend('tr', tbody);
+    createAndAppend('td', tr1, {
+      text: 'Repository: ',
+    });
+    const td2 = createAndAppend('td', tr1);
+    createAndAppend('a', td2, {
+      href: repository.html_url,
+      target: '_blank',
+      text: repository.name
+    });
+    const tr2 = createAndAppend('tr', tbody);
+    createAndAppend('td', tr2, {
+      text: 'Description:'
+    });
+    createAndAppend('td', tr2, {
+      text: repository.description
+    });
+    const tr3 = createAndAppend('tr', tbody);
+    createAndAppend('td', tr3, {
+      text: 'Forks:'
+    });
+    createAndAppend('td', tr3, {
+      text: repository.forks
+    });
+    const tr4 = createAndAppend('tr', tbody);
+    createAndAppend('td', tr4, {
+      text: 'Updated:'
+    });
+    createAndAppend('td', tr4, {
+      text: repository.updated_at
+    });
+  }
+  function renderContributors(contributors, container) {
+    container.innerHTML = '';
+    const h2 = createAndAppend('h2', container, {
+      text: 'Contributors'
+    });
+    const ul = createAndAppend('ul', container);
+
+    contributors.forEach(contributor => {
+      const li = createAndAppend('li', ul);
+      createAndAppend('a', li, {
+        href: contributor.html_url,
+        target: '_blank',
+        text: contributor.login
+      });
+      createAndAppend('img', li, {
+        src: contributor.avatar_url
+      });
+      createAndAppend('div', li, {
+        text: contributor.contributions
+      });
     });
   }
 
+  function main(url) {
+    const root = document.getElementById('root');
+    fetchJSON(url)
+      .then(repositories => {
+        repositories.sort((a, b) => a.name.localeCompare(b.name));
+        console.log(repositories);
+        const header = createAndAppend('header', root);
+        const h1 = createAndAppend('h2', header, {
```

Comment:
ESLint flags the variable `h1` as 'unused', therefore you can (and should) remove it.